osu.Game.Rulesets.Catch/UI/CatchCursorContainer.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Game.Rulesets.UI;
+
+namespace osu.Game.Rulesets.Catch.UI
+{
+    public class CatchCursorContainer : GameplayCursorContainer
+    {
+        protected override Drawable CreateCursor() => new InvisibleCursor();
+
+        private class InvisibleCursor : Drawable
+        {
+        }
```

Comment:
This is not quite what we want to have in the end - see #2901. While for relax alone it does make sense to completely hide the cursor, normally it would be desired to have it disappear after an idle period for parity with stable.

Reply:
what should the ctb cursor look like? or should i just keep the menu cursor and have that fade away via #2901 ?